extensions/wikia/Recirculation/js/views/rail.js

```diff
@@ -9,15 +9,13 @@ define('ext.wikia.recirculation.views.rail', [
 ], function ($, w, log, abTest, tracker, utils) {
 
 	var logGroup = 'ext.wikia.recirculation.views.rail',
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the declaration of the unused 'logGroup' variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "logGroup" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1854)